### ISSUE TITLE
In react native 0.69.5, gradle sync fails.

### ISSUE THREAD

**Issue by marcelocolla** [Survicate/react-native-survicate#31]
Caused by: **Could not resolve com.facebook.react:react-native:+.**

**"@survicate/react-native-survicate": "^1.1.13"**
_Type of the project:_ **expo**
_Programming language:_ **JavaScript**

Installed software:
- NDK 21.4.7075529
- Node.js 16.13.2
- Yarn 1.22.17
- npm 8.1.2
- pnpm 7.0.0
- Java 11
- Expo CLI 6.0.0
- React native 0.69.5

```bash
"Install Android SDK Build-Tools 31 (revision: 31.0.0)" complete.
"Install Android SDK Build-Tools 31 (revision: 31.0.0)" finished.
Checking the license for package Android SDK Platform 31 in /home/expo/Android/Sdk/licenses
License for package Android SDK Platform 31 accepted.
Preparing "Install Android SDK Platform 31 (revision: 1)".
"Install Android SDK Platform 31 (revision: 1)" ready.
Installing Android SDK Platform 31 in /home/expo/Android/Sdk/platforms/android-31
"Install Android SDK Platform 31 (revision: 1)" complete.
"Install Android SDK Platform 31 (revision: 1)" finished.

[stderr] FAILURE: Build failed with an exception.
[stderr] * Where:
[stderr] Build file '/home/expo/workingdir/build/node_modules/@survicate/react-native-survicate/android/build.gradle' line: 87
[stderr] * What went wrong:
[stderr] A problem occurred configuring project ':survicate_react-native-survicate'.
[stderr] > Could not resolve all files for configuration ':survicate_react-native-survicate:implementation'.
[stderr]    > Could not resolve com.facebook.react:react-native:+.
[stderr]      Required by:
[stderr]          project :survicate_react-native-survicate
[stderr]       > Cannot choose between the following variants of com.facebook.react:react-native:0.69.5:
[stderr]           - debugVariantDefaultRuntimePublication
[stderr]           - releaseVariantDefaultRuntimePublication
[stderr]         All of them match the consumer attributes:
[stderr]           - Variant 'debugVariantDefaultRuntimePublication' capability com.facebook.react:react-native:0.69.5:
[stderr]               - Unmatched attributes:
[stderr]                   - Provides com.android.build.api.attributes.BuildTypeAttr 'debug' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.category 'library' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.libraryelements 'aar' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.status 'release' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it
[stderr]           - Variant 'releaseVariantDefaultRuntimePublication' capability com.facebook.react:react-native:0.69.5:
[stderr]               - Unmatched attributes:
[stderr]                   - Provides com.android.build.api.attributes.BuildTypeAttr 'release' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.category 'library' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.libraryelements 'aar' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.status 'release' but the consumer didn't ask for it
[stderr]                   - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it
```
------------------

When applying the same settings to Expo SDK 45 application works correctly.

**Comment by sergey-duzhych** [comment]
Hi @marcelocolla. Thank you for reporting the issue. We will investigate it and will get back to you as soon as there are any updates.

**Comment by marcelocolla** [comment]
Thanks @sergey! To reproduce the error scenario, I am using Expo SDK 46 to create a new project!

**Step used:**
1. npx create-expo-app my-app
2. yarn add @survicate/react-native-survicate
3. expo prebuild --clean
4. eas build:configure (`eas-cli/1.2.0 linux-x64 node-v16.13.0`)
5. eas build --platform android --profile preview

[Repository with example](https://github.com/marcelocolla/survicate-expo-app)

**Comment by sergey-duzhych** [comment]
@marcelocolla Hi, the issue is addressed in RN library version 1.1.16. Please update and let us know if it works.

**Comment by marcelocolla** [comment]
@sergey-duzhych Hi, it's working properly :+1:! Thanks for posting the fix quickly.
